internal/census/census_job.go

```diff
@@ -61,10 +60,9 @@ func runInternal(ctx context.Context, c *censusJob) error {
 }
 
 // NextRunIn returns the duration until the next job run should be scheduled.
-// We report as ready immediately after a successful run. This doesn't mean that
-// this job will run immediately, only about as often as the configured scheduler interval.
+// Census will run every hour to ensure any interrupted jobs will be re-attempted
 func (c *censusJob) NextRunIn(_ context.Context) (time.Duration, error) {
-	return 0, nil
+	return 3600, nil
```

Comment:
```suggestion
	return 1*time.Hour, nil
```
Would doing something like the above be the more idiomatic way in Go?